tools/who_what_benchmark/tests/test_cli_image.py

```diff
@@ -65,6 +67,7 @@ def test_image_model_types(model_id, model_type, backend):
 @pytest.mark.parametrize(
     ("model_id", "model_type"),
     [
+        ("dreamlike-art/dreamlike-anime-1.0", "image-to-image"),
```

Comment:
maybe `echarlaix/tiny-random-latent-consistency` can work here instead of non-working `echarlaix/tiny-random-stable-diffusion-xl` ?

Reply:
I tried several of them including the one you mentioned and they do not work due the known bug.

Reply:
do you need torch model in these tests or openvino only enough?
We have several preconverted models here https://huggingface.co/collections/OpenVINO/image-generation-67697d9952fb1eee4a252aa8 (I requested dreamlike models too, but current status is waiting approval)

among trained but still small is https://huggingface.co/segmind/tiny-sd
but not sure that used in model scheduler currently supported

Reply:
also for speedup testing, you can expose num_infeence_steps parameter in cli (I suppose even with streigh=0.8, it runs 40 steps

Reply:
> do you need torch model in these tests or openvino only enough? We have several preconverted models here https://huggingface.co/collections/OpenVINO/image-generation-67697d9952fb1eee4a252aa8 (I requested dreamlike models too, but current status is waiting approval)
> 
> among trained but still small is https://huggingface.co/segmind/tiny-sd but not sure that used in model scheduler currently supported

Yes, I need a Torch model and I tried "[segmind/tiny-sd](https://huggingface.co/segmind/tiny-sd)" but it didn't work OOB.

And, yes `num_infeence_steps` is already exposed and I am planning to use it in the test.